templates/kotlin/base/requests/location.twig

```diff
@@ -0,0 +1,6 @@
+        return client.call(
```

Comment:
This template needs to import the `resultType` macro

Reply:
thank you !